File: examples/integrity/verify_proof/verify_proof.go
package main

import (
	"github.com/bloock/bloock-sdk-go/v2/client"
	"github.com/bloock/bloock-sdk-go/v2/entity/integrity"
)

func main() {
	integrityClient := client.NewIntegrityClient()

	var proof integrity.Proof
	_, _ = integrityClient.VerifyProof(proof)
	// if err != nil { ... }
}

File: examples/integrity/verify_proof/VerifyProof.java
import com.bloock.sdk.client.IntegrityClient;
import com.bloock.sdk.entity.integrity.Proof;

public class VerifyProof {
    public static void main(String[] args) throws Exception {
        IntegrityClient integrityClient = new IntegrityClient();

        // Proof obtained previously
        Proof proof = null;

        String root = integrityClient.verifyProof(proof);
    }
}